src/Sarif/ConsoleLogger.cs

```diff
@@ -320,7 +320,9 @@ private string FormatNotificationMessage(Notification notification)
                 throw new InvalidOperationException("Unknown notification level: " + notification.Level);
             }
 
-            var sb = new StringBuilder(issueType + " ");
+            var sb = new StringBuilder(_toolName + " : ");
+
+            sb.Append(issueType + " ");
```

Comment:
>" " [](start = 34, length = 3)

Don't we want another colon?
```
BinSkim : warning : PDB not found.
```

Reply:
For this scenario, the colon is added after the descriptor ID. You are pointing out that in the absence of a descriptor or associated rule, we will append the message without a colon. This bug has always existed. In current practice, we are not raising issues that do not have an id). Suggest I open a work item for this clean up (probably not urgent)?

---
In reply to: [316878595](https://github.com/microsoft/sarif-sdk/pull/1658#discussion_r316878595) [](ancestors = 316878595)

Reply:
No need for a work item, but my question and your answer suggest the need for a code comment.

---
In reply to: [316892439](https://github.com/microsoft/sarif-sdk/pull/1658#discussion_r316892439) [](ancestors = 316892439,316878595)

Reply:
Let me just add the colon. :) That's as easy as adding a comment or filing a bug.

---
In reply to: [316896655](https://github.com/microsoft/sarif-sdk/pull/1658#discussion_r316896655) [](ancestors = 316896655,316892439,316878595)